test-lit/semantics/omp-atomic.f90

```diff
@@ -2,6 +2,9 @@
 
 ! Check OpenMP 2.13.6 atomic Construct
 
+! RUN: %flang -fdebug-resolve-names -fparse-only OPTIONS: -fopenmp %s 2>&1
```

Comment:
Does this work? Should OPTIONS appear like this?

Reply:
I assume it is a difference in formatting with spaces again. Originally I had "only add opt if '-' in opt" , but Rich said it's too general of an assumption. Changed it to not include the opt if it has a comment character or OPTIONS in it.